Keas.Core/Domain/WorkstationAttribute.cs

```diff
@@ -0,0 +1,14 @@
+using Newtonsoft.Json;
+
+namespace Keas.Core.Domain
+{
+    public class WorkstationAttribute {
+        public int Id { get; set; }
+        [JsonIgnore]
```

Comment:
@jcubbage I think laura fixed it so we don't need this anymore?  Worth looking into later when we start actually using this part of the API